### ISSUE TITLE
No Bar to skip to specific time within VOD

### ISSUE THREAD

**Issue by JeffJeffrey12** [besuper/TwitchNoSub#17]
Not sure why, but I can't see the bar to skip to a later part in VOD properly - only arrow keys work.

**Comment by besuper** [comment]
Hi, can you send me a screenshot of your player

**Comment by JeffJeffrey12** [comment]
I just saw that it appear in fullscreen, but not when not fullscreen.
And I didn't find the option to show chat in options.

**Comment by besuper** [comment]
To enable the chat you need to click on the extension icon ![image](https://user-images.githubusercontent.com/16976439/179101340-af1114ce-9287-48d9-8c2b-ad89a6147749.png) then click on "Settings". 
You will see this: 
![image](https://user-images.githubusercontent.com/16976439/179101568-e024bf90-6b72-4be1-81a3-17e091e668c9.png)

**Comment by Rickkwa** [comment]
So here we see the player controls are appearing correctly:
![image](https://user-images.githubusercontent.com/4498693/181406917-b1c8f7b9-8497-4f22-a992-a66930e9a6e2.png)


But if I collapse the left sidebar, the player gets bigger but you can see the bottom of the stream is cut off, along with the player controls:
![image](https://user-images.githubusercontent.com/4498693/181406886-147799e2-e966-43c0-a74e-a78a496418fa.png)

**Comment by ashishkotnala29** [comment]
Issue is caused by plyr. I do not know why the switch from videojs to plyr was made but I have reverted to videojs and this issue does not happen to me.

IMO plyr is outdated and not maintained anymore and hence videojs should be preferred. I also have observed memory leaks with plyr and some UI scaling issues which are not happening after going back to videojs.

I liked the support of playing "highlights" vod so I added support for those type of vod into previous code of v3 which is based on videojs so I'm not really missing anything.
